.ci/trigger_dispatch_run.sh

```diff
@@ -37,9 +37,9 @@ pr=$(
 data=$(
   jq -n \
     --arg event_type "$dispatch_name" \
-    --arg pr_number "$(echo $pr | jq '.number')" \
-    --arg pr_sha "$(echo $pr | jq '.head.sha')" \
-    --arg pr_branch "$(echo $pr | jq '.head.ref')" \
+    --arg pr_number "$(echo "$pr" | jq '.number')" \
+    --arg pr_sha "$(echo "$pr" | jq '.head.sha')" \
+    --arg pr_branch "$(echo "$pr" | jq '.head.ref')" \
```

Comment:
Putting double quotes inside other double quotes like this requires escaping. To avoid that, please revert these changes and instead add the following comment on line 37 (directly above `data=$(`):

```bash
# shellcheck disable=SC0286
```